apps/indexer/src/api/repositories/drizzle.ts

```diff
@@ -14,14 +14,14 @@ export class DrizzleRepository {
     const query = sql`
       WITH old_data AS (
         SELECT db.average as old_amount
-        FROM dao_metrics_day_buckets db
+        FROM anticapture.dao_metrics_day_buckets db
```

Comment:
this shouldn't be added, the `view` created by ponder is a [materialized view](https://www.postgresql.org/docs/current/rules-materializedviews.html) that works the same way as a schema